posts/2021-05-11-database-ssl-primer.adoc

```diff
@@ -0,0 +1,310 @@
+---
+layout: post
+title: "Database Connections Using SSL/TLS a Primer"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/Kyle_Aure
+author_github: https://github.com/Kyle_Aure
+seo-title: Database Connections Using SSL/TLS a Primer - OpenLiberty.io
+seo-description: How to setup common database jdbc drivers (DB2, Oracle, Postgres, SQLServer) to use SSL / TLS for secure connections.
+blog_description: "How to setup common database jdbc drivers (DB2, Oracle, Postgres, SQLServer) to use SSL / TLS for secure connections."
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Database Connections Using SSL/TLS a Primer
```

Comment:
Thanks David.  I agree with the title `Database connections with TLS`.  One thing I do want to make sure of however, is that if someone searches for SSL in the blogs that this article shows up.  Since TLS and SSL are used interchangeability. Will the SEO description take care of that?

Reply:
That was my thinking- but I think `Database connections with TLS/SSL` could also work. I'm never quite sure how to handle the fact that nobody really uses SSL anymore but everyone still calls it that :)